storage/persistent-storage/persistent-storage-azure-file.adoc

```diff
@@ -0,0 +1,37 @@
+[id="persistent-storage-using-azure-file"]
+= Persistent storage using Azure File
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-azure-file
+toc::[]
+
+{product-title} supports Microsoft Azure File volumes. You can
+provision your {product-title} cluster with persistent storage using Azure.
+Some familiarity with Kubernetes and Azure is assumed.
+
+The Kubernetes persistent volume framework allows administrators to provision a
+cluster with persistent storage and gives users a way to request those
+resources without having any knowledge of the underlying infrastructure.
+Azure File volumes can be provisioned dynamically.
+
+Persistent volumes are not bound to a single project or namespace; they can be
+shared across the {product-title} cluster.
+Persistent volume claims are specific to a project or namespace and can be
```

Comment:
Persistent volume claims or PersistentVolumeClaims?

s/namespace and can be requested by users/namespace, and users can request them for _why they'd request a PVC_.